discovery-infra/tests/test_install.py

```diff
@@ -1,11 +1,22 @@
+import json
+import subprocess
+
 import pytest
 
 from tests.base_test import BaseTest
 
+
+def pytest_generate_tests(metafunc):
+    if "openshift_version" in metafunc.fixturenames:
+        cmd = "kubectl get configmap assisted-service-config -n assisted-installer -o jsonpath='{.data.OPENSHIFT_VERSIONS}'"
```

Comment:
You can get the openshift versions through with an API call to assisted-service, wouldn't that be better?